### PR TITLE
Fix/lru mem functions

### PR DESCRIPTION
**Pull request by mariotaku**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

File: src/misc/lv_lru.c [hunk 1]
```diff
@@ -82,14 +81,15 @@ lv_lru_t * lv_lru_new(size_t cache_size, size_t average_length, lv_lru_free_t *
     cache->free_memory = cache_size;
     cache->total_memory = cache_size;
     cache->seed = time(NULL);
```
Comment (kisvegabor) [inline src/misc/lv_lru.c]: `time` is not provided by every platforms, so let's use something more generic. 
```suggestion
    cache->seed = lv_rand();
```

**Review by kisvegabor — COMMENTED** [review]
Thanks, only one thing.

File: src/misc/lv_lru.c (hunk 1 above)
Comment (mariotaku) [reply src/misc/lv_lru.c]: Thanks for your suggestion. I have updated it also naming convention.

**Comment by kisvegabor** [comment]
Thank you. The renaming changes look good too. 
(I missed that this file is out of the SDL folders)

**Comment by mariotaku** [comment]
It's common if we need more cache features later. For example if we implement OpenGL based renderer, this will also be reused.

**Comment by kisvegabor** [comment]
I agree!

**Pull request merged**